src/python/install.sh

```diff
@@ -84,6 +86,9 @@ receive_gpg_keys() {
         mkdir -p "$(dirname \"$2\")"
         keyring_args="--no-default-keyring --keyring $2"
     fi
+    if [ ! -z "${KEYSERVER_PROXY}" ]; then
+	    keyring_args="${keyring_args} --keyserver-options http-proxy=${KEYSERVER_PROXY}"
```

Comment:
Can we format this and remove one tab to match the previous block?

Reply:
Good spot, interesting quirk of the github web editor there